Assets/Scripts/Lobby/LobbyManager.cs

```diff
@@ -1,14 +1,15 @@
 using System.Collections;
+using System.Linq;
```

Comment:
In game development, we want to avoid Linq like the plague. It is simply too slow.